src/test/java/org/broadinstitute/hellbender/engine/MultiVariantWalkerGroupedOnStartUnitTest.java

```diff
@@ -91,7 +91,7 @@ public void apply(List<VariantContext> variantContexts, ReferenceContext referen
     @SuppressWarnings({"unchecked"})
     public void testSimpleCaseDuplicateFile(File file, File file2) {
         final DummyExampleGroupingMultiVariantWalker tool = new DummyExampleGroupingMultiVariantWalker();
-        final String[] args = {"--variant", file.getAbsolutePath(), "--variant", file2.getAbsolutePath()};
+        final String[] args = {"--variant", file.getAbsolutePath(), "--variant", file2.getAbsolutePath(), "-R", b37_reference_20_21};
```

Comment:
How were these tests passing without -R?

Reply:
Because the walker actually handles the case where the reference context is null and will go on doing its grouping behavior properly. The reference context is needed for some of the functionality (importantly much of the functionality a tool cares about), but not the functionality I was testing here.